x-pack/filebeat/module/zeek/connection/_meta/fields.yml

```diff
@@ -0,0 +1,132 @@
+- name: connection
+  type: group
+  description: >
+    Fields exported by the Zeek conn.log
+  fields:
+    - name: destination
```

Comment:
I see you already convert all the data to ECS so these fields here should not be needed I would assume?

Reply:
Will review in full later. But this is the approach we took with Suricata as well: keep all of the original fields, since there's a lot of them, and copy out what can be mapped to ECS.

I'm perfectly good with defining all of these, and having no aliases.

Reply:
Oh, I assumed wrong. Yeah I see these are ECS fields indeed. Nicolas is right that there's no need to define the ECS fields here, as they're already defined and shared across Beats

Reply:
There are some fields that are currently not defined in ECS. I kept all the field to have coherency. The fields that are already in ECS use the same type definitions in ECS. I can remove the redundant fields if needed.

Reply:
@ruflin I will make some changes in this area, mainly about put zeek specific fields under zeek struct for now, until they are ready to be moved to the generic ecs fields.

Reply:
New commits are submitted. Please review.